src/library/Uno.Material/Styles/Application/AnimationConstants.xaml

```diff
@@ -7,4 +7,9 @@
 	<CubicEase x:Key="Material_EaseOutFunction"
 			   EasingMode="EaseOut" />
 
+	<Duration x:Key="Material_AnimationDuration">0:0:0.25</Duration>
+	<Duration x:Key="Material_AnimationDuration_Short">0:0:0.15</Duration>
+	<Duration x:Key="Material_AnimationDuration_Long">0:0:0.4</Duration>
```

Comment:
Can you try to add an explanation here on when default/short/long should be used?

Reply:
I would like to, but I don't really know I just refactored the current animations. Buttons have animation that begins at 0:0:2 and I can't tell if it's a typo or some kind of workaround.

Reply:
Yeah let's understand the begintime, can you check the git history of files and see if you can find any reason?